brainTeasers/waterJugs.md

```diff
@@ -1,3 +1,14 @@
 You have a five-quart jug, a three-quart jug, and an unlimited supply of water (but no measuring cups).
 How would you come up with exactly four quarts of water?
 Note that the jugs are oddly shaped, such that filling up exactly "half" of the jug would be impossible.
+
+
```

Comment:
Well done.. now I shall ask.. what if you can't mark the jug in any way??? What then?